courses/machine_learning/deepdive2/supplemental/solutions/question_answering_rnn.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<ul><li>"batchSize" --&gt; "batch_size"</li><li>maybe use more informative variable names for "seq_pars" and "seq_ques" as for instance "max_paragraph_length" , etc.</li><li>pep8 : spaces missing after commas in data[['question','context','text','start_word', 'end_word']]</li><li>avoid abbreviations in variable names (it makes the code harder to read). For instance: "ye" --&gt; "y_end_word". Also it seems that the naming you use for "ye" and "ys" should be reversed ("ye" should correspond to "y.end_word" and not "y_start_word". Make sure this doesn't result in a bug later on.</li><li>The name "y" here seems not informative enough. Usually, it's understood as labels - is that the case here?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Thanks for flagging the start/end bug, I had mixed them up when cleaning my code in the last steps... Fixed now!</p>